### ISSUE TITLE
Volumes __ne__ is not defined

### ISSUE THREAD

**Issue by sroet** [openpathsampling/openpathsampling#700]
Apparently `__eq__` is defined for volumes, but` __ne__` is not. This results in unexpected behavior in python 2.7.
As a minimal example:
```python
import openpathsampling as paths
s1 = paths.EmptyVolume()
s2 = paths.EmptyVolume()
print s1 == s2
print s1 != s2
```
returns
```python
True
True
```
stack overflow answer explaining it, and mentioning that this behavior is changed in python3
https://stackoverflow.com/a/30676267

Found by @singraber

**Comment by singraber** [comment]
A short survey in openpathsampling/openpathsampling
```
for i in $(find . -name "*.py"); do grep -H -n "__eq__" $i; grep -H -n "__ne__" $i; done
``` 
shows that this behavior may be also found in these classes:

```python
CallableCV in collectivevariable.py
Ensemble in ensemble.py
SampleSet in sample.py
```
They all have a `__eq__()` function implemented but not the `__ne()__`.

**Comment by dwhswenson** [comment]
Thanks for catching this. I'll try to put together a quick PR on this (unless @jhprinz tells me he's already working on it.)

**Comment by dwhswenson** [comment]
Closed by #702.
